examples/siamese/README.md

```diff
@@ -0,0 +1,115 @@
+This example follows the Caffe [Siamese Tutorial](http://caffe.berkeleyvision.org/gathered/examples/siamese.html) and shows
+how to use DIGITS to teach a neural network to map an image from the MNIST dataset to a 2D point, while trying to minimize the distance between points of the same class and maximize the distance between points of different classes.
+
+## Dataset Creation
+
+In our dataset, each input sample is a pair of images and the associated label is set to `1` when both images are from the same class (i.e. both images represent the same digit) and `0` when images are from different classes.
+
+Since DIGITS expects input data to be images, we will create one RGB image for each pair of (grayscale) images in the MNIST
+dataset:
+- the Blue channel will contain the first image in the pair
+- the Green channel will contain the second image in the pair
+- the Red channel will be unused and set to 0.
+
+If you followed the introductory walkthrough on [GettingStarted](../../docs/GettingStarted.md), you should have the MNIST dataset created in DIGITS already and DIGITS will have conveniently created a list of MNIST image paths and stored it in a file called `train.txt`.
+
+Click on your MNIST image classification model to display dataset information: this will allow you to find the path to the job directory which includes the `train.txt` file:
+
+![MNIST image classification dataset](mnist-classification-dataset.png)
+
+In this example the path is `/fast-scratch/gheinrich/ws/digits/digits/jobs/20151111-210842-a4ec`.
+
+Non-classification datasets may be created in DIGITS through the "other" type of datasets. For these datasets, DIGITS expects the user to provide a set of LMDB databases.
+Note that since labels may be vectors (or matrices), it is not possible to use a single LMDB database to hold the image and its label. Therefore DIGITS expects one LMDB database for the images and a separate LMDB database for the labels.
+
+The first step in creating the dataset is to create the LMDB databases. In this example you will use the Python test script located in `/examples/siamese/create_db.py`.
+
+To create a train database of 100000 pairs of images into a folder called `siamesedb`:
+```sh
+$ cd $DIGITS/examples/siamese
+$ create_db.py siamesedb ../../digits/jobs/20151111-210842-a4ec/train.txt  -c 100000
+```
+The script also creates a validation database of 1000 samples. Overall, the script creates:
+- train image and label databases,
+- validation image and label databases,
+- train and validation mean images,
+- two test images: one for a pair of images from the same class, and one for a pair of images from different classes.
+
+Now that we have created the required files, we may create the dataset using DIGITS. On the main page, click `New Dataset\Images\Other`:
```

Comment:
This word-wraps strangely. How about `New Dataset > Images > Other` instead? (with spaces)